pkg/kubelet/dockershim/network/testing/mock_network_plugin.go

```diff
@@ -30,106 +30,127 @@ import (
 	"k8s.io/kubernetes/pkg/kubelet/dockershim/network"
 )
 
-// Mock of NetworkPlugin interface
+// MockNetworkPlugin is the Mock of NetworkPlugin interface
 type MockNetworkPlugin struct {
 	ctrl     *gomock.Controller
-	recorder *_MockNetworkPluginRecorder
+	recorder *MockNetworkPluginRecorder
```

Comment:
Does this file actually need to be updated? It's generated code, so these changes might end up getting wiped out? The docstrings certainly won't get generated and I think the _-prefaced interface is intentional.